docs/setup/windows.md

```diff
@@ -19,11 +19,15 @@ Alternatively, you can also download a [Zip archive](/docs/?dv=winzip), extract
 
 >**Tip:** Setup will add Visual Studio Code to your `%PATH%`, so from the console you can type 'code .' to open VS Code on that folder. You will need to restart your console after the installation for the change to the `%PATH%` environmental variable to take effect.
 
-## User setup versus system setup
+## User vs. System setup
```

Comment:
It's a best practice to not change H2 headers as people might have created book marks to the section.
Unless anyone objects, I'll revert this change